app/game/field_outputter.py

```diff
@@ -1,24 +1,70 @@
 
 from app.slack import slacker
+from app.game.field import Object, Bomb, Fire, Item, Person
+
+object_emoji_map = {
+    Object.wall: ":black_square_for_stop:",
+    Object.empty: ":white_large_square:",
+    Object.block: ":new_moon_with_face:"
+}
+
+item_emoji_map = {
+    Item.add_bomb: ":b:",
+    Item.fire: ":fire:",
+    Item.speed: ":ice_skate:"
+}
+
+person_emoji_map = {
+    0: ":alien:",
+    1: ":rage:",
+    2: "innocent",
+    3: ":smiling_imp:"
+}
+
+
+def object_to_emoji(object):
+    if isinstance(object, Bomb):
+        return ":bomb:"
+
+    if isinstance(object, Fire):
+        return ":boom:"
+
+    if isinstance(object, Item):
+        return item_emoji_map[object]
+
+    if isinstance(object, Person):
+        return person_emoji_map[object.num]
+
+    return object_emoji_map[object]
+
 
 class FieldOutputter:
     recent_field_ts = {}
 
     @classmethod
     def post_field(cls, channel, field, new_message=False):
-        field_text = "TODO: create this by field"
+
+        field_text = ""
+        for y in range(0, field.y_size):
```

Comment:
A first argument of range function can eliminate.
If range function called with a single argument, it means 0 to {first_argument} iterator